include/conf/find.hpp

```diff
@@ -10,6 +10,17 @@ namespace color_coded
   {
     namespace fs = boost::filesystem;
 
+    inline fs::directory_iterator find_file(fs::path const &dir, std::string const &file)
+    {
+      fs::directory_iterator const end;
```

Comment:
Oh, I thought you were referring to the function. I also made the directory_iterator in find() static as well.